resty/src/App.test.js

```diff
@@ -1,9 +1,38 @@
-import React from 'react';
-import { render } from '@testing-library/react';
-import App from './App';
-
-test('renders learn react link', () => {
-  const { getByText } = render(<App />);
-  const linkElement = getByText(/learn react/i);
-  expect(linkElement).toBeInTheDocument();
+import React from 'react'
+import { render, fireEvent, waitFor, screen } from '@testing-library/react'
+import '@testing-library/jest-dom/extend-expect'
+import App from '../app-number.js'
```

Comment:
Now that you have pulled everything out of that resty folder, this file can't be found